cypress/integration/ContextMenu.spec.ts

```diff
@@ -0,0 +1,260 @@
+        cy.findByText('Inbox').should('not.exist');
+      });
+
+      it('should close submenu when moving pointer from open submenu to enabled or disabled trigger siblings', () => {
+        // To submenu and then to sibling trigger in parent
+        pointerOver('Bookmarks →');
+        pointerOver('Inbox');
+        pointerOver('Tools →');
+        cy.findByText('Inbox').should('not.exist');
+
+        // To submenu then to a disabled trigger sibling in parent
+        pointerOver('Bookmarks →');
+        pointerOver('Inbox');
+        pointerOver('History →');
+        cy.findByText('Inbox').should('not.exist');
+      });
+
+      it('should close unassociated submenus when moving pointer back to the root trigger', () => {
+        // Open multiple nested submenus and back to trigger in root menu
+        pointerOver('Bookmarks →');
+        pointerOver('Modulz →');
+        pointerOver('Radix');
+        pointerOver('Bookmarks →');
+
+        cy.findByText('Inbox').should('be.visible');
+        cy.findByText('Radix').should('not.exist');
+      });
+
+      it('should close all menus when clicking item in any menu, or clicking outside', () => {
+        // Root menu
+        cy.findByText('New Tab').click();
+        cy.findByText('New Tab').should('not.exist');
+
+        // Submenu
+        cy.findByText('Right Click Here').rightclick();
+        pointerOver('Bookmarks →');
+        cy.findByText('Inbox').click();
+        cy.findByText('New Tab').should('not.exist');
+        cy.findByText('Inbox').should('not.exist');
+
+        // Click outside
+        cy.findByText('Right Click Here').rightclick();
+        cy.get('body').click({ force: true });
+        cy.findByText('New Tab').should('not.exist');
+      });
+    });
+
+    describe('When using keyboard', () => {
+      it('should not open submenu when moving focus to trigger', () => {
+        cy.findByText('Bookmarks →').focus();
+        cy.findByText('Inbox').should('not.exist');
+      });
+
+      it('should open submenu and focus first item when pressing right arrow, enter or space key', () => {
+        function shouldOpenOnKeydown(key: string) {
+          cy.findByText('Bookmarks →').trigger('keydown', { key });
+          cy.findByText('Inbox').should('be.focused').trigger('keydown', { key: 'ArrowLeft' });
+        }
+
+        shouldOpenOnKeydown(' ');
+        shouldOpenOnKeydown('Enter');
+        shouldOpenOnKeydown('ArrowRight');
+      });
+
+      it('should close only the focused submenu when pressing left arrow key', () => {
+        cy.findByText('Bookmarks →').type('{enter}');
+        cy.findByText('Modulz →').type('{enter}');
+        cy.findByText('Stitches').type('{leftarrow}').should('not.exist');
+        cy.findByText('Modulz →').should('be.visible');
+        cy.findByText('New Window').should('be.visible');
+      });
+
+      it.only('should close all menus when pressing escape, enter or space key on any item', () => {
```

Comment:
`it.only` means the other tests aren't running right?

Reply:
Double whoops, good spot!